certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1,1411 +1,1422 @@
+        },
+        "summary": "BookBuilder - Get Template List",
+        "operationId": "get-template-list",
+        "parameters": [],
+        "description": "A template is a predefined list of content to be compiled in a PDF. This end point works without any parameters and retrieves the list of templates available for the user. Templates need to defined/created in FactSet workstation."
+      }
+    },
+    "/book-builder-api/v1/create-book-from-template": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "bookName": {
+                  "type": "string",
+                  "description": "bookName"
+                },
+                "bookID": {
+                  "type": "string",
+                  "description": "bookID"
+                }
+              }
+            }
+          }
+        },
+        "summary": "BookBuilder - Create Book from Template",
+        "description": "This endpoint retrieves book status, book name, and book ID for ticker requested in JSON format. This end-point accepts ticker and template_id as inputs. If the template_id input is not used, a book will be created with FactSet's default template.",
+        "operationId": "create-book-from-template",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ticker": {
+                  "type": "string",
+                  "description": "ticker"
+                },
+                "template_id": {
+                  "type": "string",
+                  "description": "template_id"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/book-builder-api/v1/download-api-book/{book_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "summary": "BookBuilder - Get PDF",
+        "description": "This endpoint will return the PDF output given a book_id.",
+        "operationId": "get-pdf",
+        "parameters": [
+          {
+            "name": "book_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Book ID",
+            "description": "ID of the book to download"
+          }
+        ]
+      }
+    },
+    "/content/symbology/v3/identifier-resolution": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@KyleVErwin empty schema is not allowed. please define a proper schema. Please fix all empty schemas in the swagger file.